### DISCUSSION TITLE
Highlight area and object resizing features

### DISCUSSION THREAD

**Discussion by urbanskalar** [flameshot-org/flameshot#3357] [Ideas]
Hi all!

Recently I discovered flameshot and I love it so far. Sadly there are a few things that I use on a daily basis and are preventing me to fully switch to this program. One of them is the lack of the area highlight feature. I know there is marker tool, which is very useful for text, but I very often need to highlight area of the screen. Attached is the screenshot example from some other program that shows the use of such tool.

![image](https://github.com/flameshot-org/flameshot/assets/91073365/d9256185-4cf8-47bb-99aa-dbc8d9121eaa)

Another thing is the resizing and object moving feature. For example when I draw a square, sometimes I want to adjust its position or change its size.

Before I posted the suggestions I did some googling to check if any of this is already implemented. I didn't find any answers. That's why I assume it isn't. I am not using flameshot long enough to be certain. In case this is already supported please let me know how to use this features and I apologize for spamming here.

**Comment by mmahmoudian** [comment]
Thanks for the feature request, but this is duplicate of #249 and #1498
